shipyard_blueprints/portable/shipyard_portable/cli/trigger.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+import sys
+import time
+import shipyard_bp_utils as shipyard
+from shipyard_templates import ShipyardLogger, ExitCodeException, Etl
+from shipyard_portable import PortableClient
+from shipyard_bp_utils.artifacts import Artifact
+
+logger = ShipyardLogger.get_logger()
+INTERVAL = 60
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--access-token", dest="access_token", required=True)
+    parser.add_argument("--flow-id", dest="flow_id", required=True)
+    parser.add_argument(
+        "--wait-for-completion",
+        dest="wait_for_completion",
+        required=False,
+        default="TRUE",
+    )
+
+    return parser.parse_args()
+
+
+def main():
+    try:
+        args = get_args()
+        flow_id = args.flow_id
+        portable = PortableClient(args.access_token)
+        portable.connect()
+        flow_response = portable.trigger_sync(args.flow_id)
+
+        wait = shipyard.args.convert_to_boolean(args.wait_for_completion)
+        flow_status = portable.get_sync_status(flow_id)
+        flow_disposition = flow_status["disposition"]
+        exit_code = portable.determine_sync_status(flow_status)
+        if wait:
+            logger.info("The flow will be checked every 60 seconds until completion")
```

Comment:
logger.info("The flow will be checked every 60 seconds until completion")
should be             logger.info(f"The flow will be checked every{INTERVAL} seconds until completion")